src/Integration.Vsix.UnitTests/SonarLintTagger/TaggerProviderTests.cs

```diff
@@ -126,7 +126,6 @@ public void CreateTagger_should_create_tracker_when_analysis_is_supported()
             tagger.Should().NotBeNull();
 
             VerifyCheckedAnalysisIsSupported();
-            VerifyAnalysisWasRequested();
```

Comment:
This is testing behaviour of the `TextBufferIssueTracker`.
It's tested in the `TextBufferIssueTrackerTests`, and it can't easily be tested here now as the request is no longer processed synchronously.